tools/gen_translations.py

```diff
@@ -71,7 +71,37 @@
 CFName = namedtuple('CFName', 'standard_name long_name units')
 """
 
-HEADER_GRIB = """
```

Comment:
From what I can see, there is no need to duplicate this chunk of text, because the change to "name=" when writing the grib translations file is all that was needed.  So on the basis of DRY, I'd keep this as it was.